src/PowerShellEditorServices/Services/Symbols/Visitors/AstOperations.cs

```diff
@@ -136,7 +140,7 @@ static AstOperations()
                         commandCompletion.ReplacementLength)
                     : null,
                 commandCompletion.CompletionMatches.Count);
-
+            }
             return commandCompletion;
```

Comment:
Since it now returns when not null, the `stopwatch` is superfluous and the the `LogTrace` is never triggered. Now, I don't really care about the trace log or the stopwatch, so I propose we just remove them. @SeeminglyScience what do you think about that?

Reply:
@dkattan I am happy to take this PR we just needed to fix this bit up. Did you want to do that? Otherwise I can get to it eventually.

Reply:
I will handle it so I can get this in.

Reply:
Thanks!

Reply:
Once i had it in my editor I saw what was going on 😄